install/db/dbmigrate_copyright-author.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/***********************************************************
+ Copyright (C) 2017 Siemens AG
+ 
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ version 2 as published by the Free Software Foundation.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ ***********************************************************/
+
+
+/**
+ * \brief reinsert the old email/url/author to new author table
+ * and delete the same from copyright table
+ */
+
+
+  $countAuthorColumns = $dbManager->getSingleRow("SELECT count(*) FROM copyright WHERE type <> 'statement' AND content IS NOT NULL;",array(),__METHOD__);
```

Comment:
Maybe we should use `!=` instead of `<>` because it clearly reads `not equal`?
I had to look up, what `<>` does and both are supported. What do you think?